tasks/self-introduction/self-introduction.md

```diff
@@ -1,59 +1,59 @@
 # Self-introduction
 
-Задача — записать короткое (**3–5 минут**) видео с самопрезентацией. Расскажите о себе, о том, почему вы заинтересовались программированием, как и с какой целью пришли в RS School. В качестве референса можно использовать выполненное вами задание [Markdown & Git](https://rolling-scopes-school.github.io/stage0/#/stage0/tasks/git-markdown).
+The goal is to record a short (**3-5 minutes**) self-presentation video. Tell us about yourself, why you got interested in programming, how and for what purpose you came to RS School. As a reference, you can use the task you completed [Markdown & Git](https://rolling-scopes-school.github.io/stage0/#/stage0/tasks/git-markdown).
 
-> ## О чём рассказать
+> ## What to tell about
 >
-> - О том, кто вы такой. Имя, основной навык и роль. Например, "My name is ... and I'm a junior frontend developer".
-> - Вкратце об образовании. Например, "I am a recent computer science graduate from Belarusian State University of Informatics and Radioelectronics and am currently attending the RSSchool frontend course".
-> - О том, что вам интересно изучать, о профессиональных планах и о своей мотивации заниматься программированием — в том числе это поможет менторам в выборе самых мотивированных студентов, попавших в wait list.
-> - О вашем учебном или пет-проекте, над которым вы работали в последнее время. Опишите его цель, основные технологии, сложности, с которыми пришлось столкнуться, и результаты.
+> - Tell us about who you are. Name, primary skill and role. For example, "My name is ... and I'm a junior frontend developer".
+> - Tell us briefly about your education. For example, "I am a recent computer science graduate from Belarusian State University of Informatics and Radioelectronics and I am currently attending the RSSchool frontend course".
+> - Tell us about what you are interested in studying, about your professional plans and about your motivation to do programming. This will help mentors in choosing the most motivated students from the wait list.
+> - Tesll us about a project you've been working on recently. Describe its purpose, core technologies, challenges encountered, and results.
```

Comment:
Misspelling - Te**s**ll or it should be Tesla :)

Tell us about projects you've been working on during the course